15-draft.json

```
@@ -1151,7 +1209,12 @@
           "properties": {
             "type": {
               "description": "Type of the feed, for example <samp>rss</samp>, <samp>atom</atom>, <samp>ical</samp>",
```

```suggestion
              "description": "Type of the feed",
```

Remove the "for example" here?